src/app/containers/model/model.container.ts

```diff
@@ -156,6 +156,9 @@ export class ModelComponent implements OnInit {
     // triggered only when a file is uploaded
     fileUpload(file: File): void {
         this.fileNotChosen = undefined;
+        if ( file.name !== this.uploadedFileName ) {
```

Comment:
What happens if someone realizes that they just uploaded the wrong version of a file the first time and wants to reupload it? I guess I'm wondering if we even need this check there at all or just call `resetImage` every time.